### PR TITLE
Probe wizard move to

### PR DESCRIPTION
**Pull request by wmariz**
### Description

Add setting to move to specific position on PROBE_OFFSET_WIZARD

### Benefits

Add setting to move to specific position on PROBE_OFFSET_WIZARD

### Configurations

[Configuration_adv.zip](https://github.com/MarlinFirmware/Marlin/files/5551239/Configuration_adv.zip)

### Related Issues

#19650

### CODE REVIEW

File: Marlin/Configuration_adv.h [hunk 1]
```diff
@@ -1085,6 +1085,8 @@
     //#define PROBE_OFFSET_WIZARD
     #if ENABLED(PROBE_OFFSET_WIZARD)
       #define PROBE_OFFSET_START -4.0   // Estimated nozzle-to-probe Z offset, plus a little extra
+      //#define PROBE_OFFSET_MOVE_TO_X (X_BED_SIZE / 2) // uncomment if you need to move to specific X position after homing
```
Comment (thisiskeithb) [inline Marlin/Configuration_adv.h]: `X_CENTER` would be better here.

File: Marlin/Configuration_adv.h [hunk 2]
```diff
@@ -1085,6 +1085,8 @@
...
+      //#define PROBE_OFFSET_MOVE_TO_Y (Y_BED_SIZE / 2) // uncomment if you need to move to specific Y position after homing
```
Comment (thisiskeithb) [inline Marlin/Configuration_adv.h]: Similarly, `Y_CENTER` would be better here.

**Comment by qwewer0** [comment]
This doesn't seem to work for me. Ender 3, BLTouch, SKR Mini E3 v1.2

I don't think a `G0`/`G1` call is the right way to move the nozzle to the desired position. While I wasn't able to make `do_blocking_move_to_xy` to work in `menu_probe_offset.cpp` (always got stepper motor siren), I still think that or some other function should be called instead of a `G0`/`G1` move.

If it would needs to be a `G0`/`G1` call, then I think it should look something like this instead:
```
  #if ENABLED(Z_SAFE_HOMING)
    queue.enqueue_one_now(PSTR("G0 X" STRINGIFY(Z_SAFE_HOMING_X_POINT) " Y" STRINGIFY(Z_SAFE_HOMING_Y_POINT) " F" STRINGIFY(HOMING_FEEDRATE_XY)));
  #else
    queue.enqueue_one_now(PSTR("G0 X" STRINGIFY(X_CENTER) " Y" STRINGIFY(Y_CENTER) " F" STRINGIFY(HOMING_FEEDRATE_XY)));
  #endif
```

It does not make sense to me to have `PROBE_OFFSET_MOVE_TO_X`/`Y`, because if someone has the `Z_SAFE_HOMING` set to a different position then `PROBE_OFFSET_MOVE_TO_X`/`Y` would need to be set the same to make the measurement accurate, and so `PROBE_OFFSET_MOVE_TO_X`/`Y` would be obsolete. So instead, we could just use the `Z_SAFE_HOMING` position if it is enabled and use `X`/`Y_CENTER` otherwise.

**Comment by thinkyhead** [comment]
I've modified the code to follow the methodology of other menu items that want to home or do a move, so it should behave better.

**Comment by qwewer0** [comment]
@thinkyhead, @wmariz 
When I go to Probe Offset Wizard in the LCD

1. It shows "Homing XYZ"
2. Before homing is done the LCD goes back to the status screen
3. After it homed, it doesn't move to bed center (`PROBE_OFFSET_WIZARD_XY_POS`), stays at homing position
4. If I select Probe Offset Wizard again, LCD shows "Moving..."
5. It starts to move to home direction (X/Y min) until it hits one of the endstops
6. After it hit an endstop it starts to move to X/Y max and crashes into the frame (at this point I shot down the printer)

Ender 3, SKR Mini E3 v1.2, Latest Bugfix
[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/5558118/Configuration.zip)

If needed, I could open an issue for this, but I hope that it can be solved with some easy fix.

**Pull request merged**